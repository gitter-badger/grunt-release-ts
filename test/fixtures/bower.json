{
  "name": "grunt-release-ts-test-bower",
  "version": "0.0.1.who.care.I.will.take.component.version.anyway"
}