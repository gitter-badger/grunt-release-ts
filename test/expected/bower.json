{
  "name": "grunt-release-ts-test-bower",
  "version": "0.0.13"
}
